firmware/hw_layer/ports/stm32/stm32f7/mpu_util.cpp

```diff
@@ -186,8 +186,8 @@ void stm32_stop() {
 	enginePins.warningLedPin.setValue(0);
 
 	// Do anything the board wants to prepare for stop mode - enabling wakeup sources!
-	boardPrepareForStop();
 
+	boardPrepareForStop();
```

Comment:
huh?

Reply:
Commented in Slack. Accidently just moved that line down while comparing files. Does not need changing, and in reality nothing changed.

Reply:
can you undo the change to reflect that nothing changed?

Reply:
line got deleted still